src/Snackbar/Snackbar.d.ts

```diff
@@ -21,7 +22,7 @@ export interface SnackbarProps
   onMouseLeave?: React.MouseEventHandler<any>;
   open: boolean;
   resumeHideDuration?: number;
-  SnackbarContentProps?: Object;
+  SnackbarContentProps?: SnackbarContentProps;
```

Comment:
This currently breaks the test here:

https://github.com/mui-org/material-ui/blob/e9e8e5d6045a85374454b380b99b507878f2e05f/test/typescript/components.spec.tsx#L568-L570

but that should be remedied once https://github.com/DefinitelyTyped/DefinitelyTyped/pull/22582 is merged.

Reply:
Merged.